x-pack/plugins/infra/server/lib/alerting/common/messages.ts

```diff
@@ -271,3 +271,18 @@ export const tagsActionVariableDescription = i18n.translate(
     defaultMessage: 'List of tags associated with the entity where this alert triggered.',
   }
 );
+
+export const originalAlertStateActionVariableDescription = i18n.translate(
+  'xpack.infra.metrics.alerting.originalAlertStateActionVariableDescription',
+  {
+    defaultMessage: 'The original alert state. This is only available in the recovery context',
```

Comment:
Is it obvious to the user what the "original alert state" is? Do we want to indicate that this is the alert state that this alert recovered from?

Reply:
Updated the descriptions, let me know what you think?